src/core/public/chrome/ui/header/header_breadcrumbs.tsx

```diff
@@ -42,7 +42,18 @@ interface Props {
 
 export function HeaderBreadcrumbs({ appTitle$, breadcrumbs$ }: Props) {
   const appTitle = useObservable(appTitle$, 'OpenSearch Dashboards');
-  const breadcrumbs = useObservable(breadcrumbs$, []);
+  // const breadcrumbs = useObservable(breadcrumbs$, []);
```

Comment:
Why do we change chrome header?

Reply:
We need to add workspace overview page to breadcrumb for pages when in a workspace.
https://github.com/opensearch-project/OpenSearch-Dashboards/pull/6584/files/90bffd3f239f2d327f46556874ff1b8ce245d493#diff-9dde91ac5243d0871960bc1cc6f099b60a4c651eb08fd3e0dff703dd2910a8b7

Reply:
why can't it just subscribe to `breadcrumbs$`?

Reply:
In that case, we should intercept `setBreadcrumbs` method to prepend workspace overview page instead of modifying the consumer parts I think?

Reply:
this logic has been moved to workspace plugin